lib/ar_lazy_preload/active_record/collection_association.rb

```diff
@@ -16,6 +16,6 @@ def ids_reader
         @association_ids ||= reader.map(&primary_key)
       end
     end
-    # rubocop:enable Metrics/AbcSize
+    # rubocop:enable Metrics/AbcSize, Metrics/CyclomaticComplexity
```

Comment:
Looks like we do not need to disable `Metrics/CyclomaticComplexity` here

Reply:
![image](https://user-images.githubusercontent.com/33079237/87943454-a47ec080-cad0-11ea-8f57-fdd311cc7d48.png)
Remove `Metrics/CyclomaticComplexity`, rubocop will discover an offense.
You can see https://github.com/rubocop-hq/rubocop/pull/8149